### ISSUE TITLE
Project Proposal Feedback

### ISSUE THREAD

**Issue by vikizzz** [COGS118A/Group016-Wi23#1]
Total: 5.4

Abstract (0.5/1)

You should write this in pragraphs format, as the instructions required. It also doesn't make sense to use a completely different dataset to guide your initial implementation AND THEN use your actual dataset; instead you can consider using a smaller version of your dataset. Why use two? You need a justification.

Background (0.5/1)

If your primary dataset is going to be CIFAR-100, then this is an insufficient background section as there are a plethora of benchmark models out there. You should give a background of the dataset that you choose to use to solve a problem. The background needs to be about the problem, not only how you solve it. Its more of a context or history + why you are using that dataset and what has been done with it previously.

Research Problem(& Significance, Purpose of the project) (0.5/1.5)

We're confused here, what are you solving? Are you just going to train one of these typical ML datasets and measure accuracy? This has been done countless times. What happened to the moth and butterfly dataset? Why use two dataset?

If your whole project is centered around classifciation performance on image classification datasets, you need to use a Convolutional Neural Network (CNN). At the very least, as a comparison to the other model approaches. Your background section should also be amended to include the relevant background of how these datasets have been classified with other models in the past, their performance, and a benchmark model for you to compare your results to. You are solving a problem that has already been done countless times, so if you want to continue on this project direction you should provide the necesary background and implementation to make it substantial for your group of 4.

How would your work "improve their accuracies" and "expand their applicability in real-world scenarios"? 

Data (1.0/1.5)

You mention CIFAR-100 in your abstract, and moth and butterflies, but then you are using CIFAR-10 here. Be more clear. Are you sure the data is preprocessed? If it is, then how would you have preprocessed it? I am concerned that this project is not enough with the current scope.

Proposed Solution (1.0/1.5)

You don't need to mention loading data in your solution. You also don't provide enough detail to demonstrate that the solution is possible, which is obviously is because its been done. Why do you use PCA? What is PCA? Why is it helpful here? You need to convince me and yourself that you're not just copying this from the internet. Try to explain it and understand it.

For your project, you need to use more models and compare the results. The current scope is insufficient. 

Metrics (1.0/1.5)

Why use harmonic mean? What is precision here? What is recall here? Why is an equal weighting appropriate? Why not do something with class imbalance, like a weighted loss function? Using f1-score as a metric does not solve the algorithmic challenges of class imbalance. That's an evaluation metric, not a training objective, so the model still has to handle class imbalance.

Ethics & Privacy (0.5/0.5)

Team Expectations (0.25/0.25)

Timeline (0.15/.25)
Dates were changed but fake names are still there. Clearly didn't finish editting this.

Other Comments - Your project scope is insufficient, you need make the changes here going forward. If you make all of these changes, you will get the points deducted here back. Also: Delete the instructions! Don't include the instructions in your writeup.

@stephenjarrell19
